src/storage/sql_storage.cc

```diff
@@ -481,17 +481,14 @@ std::vector<std::shared_ptr<CdsObject>> SQLStorage::browse(const std::unique_ptr
     std::shared_ptr<SQLResult> res;
     std::unique_ptr<SQLRow> row;
 
-    bool haveObjectType = false;
-
-    if (!haveObjectType) {
+    {
```

Comment:
Shouldn't this block be removed instead?

Reply:
I tried that. There's some variable redefinition stuff going on. Easiest to just keep it.

Reply:
That makes it more worrying!

Reply:
Removed. Fixed the variable redefinition later on by clearing it before use.